generator/README.md

```diff
@@ -86,7 +86,7 @@ auths:
     version: 2  # SNMP version to use. Defaults to 2.
                 # 1 will use GETNEXT, 2 and 3 use GETBULK.
 
-    # Community string is used with SNMP v1 and v2. Defaults to "public_v2".
+    # Community string is used with SNMP v1 and v2. Defaults to "public".
     community: public_v2
```

Comment:
This example default is wrong as well. This should be just `public`.

Reply:
Fixed at the same time I've rebased this PR to fix conflict.